src/test/java/executor/service/model/ThreadPoolConfigDTOTest.java

```diff
@@ -0,0 +1,37 @@
+package executor.service.model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ThreadPoolConfigDTOTest {
+
+    @Test
+    public void testConstructorAndGetters() {
+        Integer corePoolSize = 5;
+        Long keepAliveTime = 1000L;
+        ThreadPoolConfigDTO config = new ThreadPoolConfigDTO(corePoolSize, keepAliveTime);
+        assertEquals(corePoolSize, config.getCorePoolSize());
+        assertEquals(keepAliveTime, config.getKeepAliveTime());
+    }
+
+    @Test
+    public void testSetters() {
+        ThreadPoolConfigDTO config = new ThreadPoolConfigDTO();
+        config.setCorePoolSize(10);
+        config.setKeepAliveTime(500L);
+        assertEquals(Integer.valueOf(10), config.getCorePoolSize());
+        assertEquals(Long.valueOf(500L), config.getKeepAliveTime());
```

Comment:
May I suggest these small changes? :)
```suggestion
        Integer corePoolSize = 10;
        Long keepAliveTime = 500L;
        config.setCorePoolSize(corePoolSize);
        config.setKeepAliveTime(keepAliveTime);
        assertEquals(corePoolSize, config.getCorePoolSize());
        assertEquals(keepAliveTime, config.getKeepAliveTime());
```